src/mlx_omni_server/chat/mlx/prompt_cache.py

```
@@ -25,78 +58,85 @@ class PromptCache:
-        prompt_cache.tokens.extend(result_prompt)
-        return result_prompt, cached_tokens
+    cached_token_count: int = 0
+
+    def reset_prompt_cache(self, current_model_id, current_model, prompt):
+        """Resets the prompt cache and associated state.
+
+        Args:
+            prompt (List[int]): The tokenized new prompt which will populate the
+                reset cache.
+        """
+        logger.debug(f"*** Resetting cache. ***")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)